### DISCUSSION TITLE
im using laravel orchid for a project with crud operations so i created a button to add a new row in a table but cant seem to add another to delete in the function

### DISCUSSION THREAD

**Discussion by ghost** [orchidsoftware/platform#2391] [General]
public function commandBar(): iterable { return [    
      Link::make('Create new')
        ->icon('pencil')
        ->route('platform.contact.edit')
     ];
  
want to add a remove function inside the same function how do i do that

**Comment by tabuna** [comment]
Hi, @farirai123 Unfortunately the question seems strange. Can you elaborate on what you are trying to achieve? To add a new button, you need to specify it in an array:
```php
public function commandBar(): iterable { 
    return [    
      Button::make('Create new')->...,
      Button::make('Other button')->...
    ];
}
```

**Reply by ghost** [reply]
manage to fixed it by adding another buttor and separating the two buttons using a comma like what you did thank

**Reply by ghost** [reply]
this is another problem im facing can you help

https://laracasts.com/discuss/channels/laravel/am-getting-errors-while-trying-to-send-mail-in-laravel-using-smtp

**Comment by ghost** [comment]
can you kindly help me i am using mailgun to send mail from laravel orchid
documentation but getting this error

below is the link to the laracasts question
https://laracasts.com/discuss/channels/laravel/am-getting-errors-while-trying-to-send-mail-in-laravel-using-smtp
